code/game/machinery/doors/door.dm

```diff
@@ -109,6 +109,10 @@ var/list/all_doors = list()
 
 	if(allowed(user))
 		open()
+	else if(isanimal(user))
+		var/mob/living/simple_animal/SA = user
+		if(SA.can_force_doors)
+			open()
```

Comment:
I'm changing it on airlocks right now but the base `door` type applies to non-airlocks as well, so this block will stay as is